src/style-installer.js

```diff
@@ -159,12 +159,14 @@ export function makeBodyVisible(doc, opt_waitForExtensions) {
       }
     }
   };
-  waitForBody(doc, () => {
-    const perf = performanceFor(doc.defaultView);
+  const win = doc.defaultView;
+  const docState = documentStateFor(win);
+  docState.onBodyAvailable(() => {
```

Comment:
This has to continue to use `waitForBody` from DOM because this code reacts to arrows where none of the services could have been installed yet.